File: Python/1-Important-Programs/09-prime-numbers-for-given-range.py
num1 = int(input("Enter start number: "))
num2 = int(input("Enter end number: "))
ans = []

for i in range(num1, num2+1):
    flag = 0

    if i < 2:
        continue
    if i == 2:
        ans.append(2)
        continue

    for j in range(2, i):
        if i%j == 0:
            flag = 1
            break
    
    if flag == 0:
        ans.append(i)

print("The Prime Numbers for given range is: ", ans)

File: Java/1-Important-Programs/09-prime-numbers-for-given-range.java
import java.util.Scanner;

class Main {
    public static void main(String[] args) {
        Scanner getInput = new Scanner(System.in);

        System.out.println("Enter start number:");
        int num1 = getInput.nextInt();

        System.out.println("Enter end number:");
        int num2 = getInput.nextInt();

        System.out.println("\n");

        for (int i = num1; i < num2; i++) {
            boolean flag = false;
            if (i < 2) {
                continue;
            } else if (i == 2) {
                System.out.println(2);
                continue;
            } else {
                for (int j = 2; j < i; j++) {
                    if (i % j == 0) {
                        flag = true;
                        break;
                    }
                }
            }
            if (flag == false) {
                System.out.println(i);
            }
        }
        getInput.close();
    }
}